src/features/fireWeather/components/DailyModelsDisplay.tsx

```diff
@@ -68,22 +68,25 @@ const DailyModelsDisplay = ({ values }: Props) => {
               <TableRow>
                 <TableCell>Wind Dir</TableCell>
                 {values.map(v => (
-                  <TableCell key={v.datetime}>{Math.round(v.wind_direction)}</TableCell>
+                  <TableCell key={v.datetime}>
+                    {v.wind_direction != null && Math.round(v.wind_direction)}
+                  </TableCell>
                 ))}
               </TableRow>
               <TableRow>
                 <TableCell>Wind Spd (km/h)</TableCell>
                 {values.map(v => (
                   <TableCell key={v.datetime}>
-                    {v.wind_speed.toFixed(MODEL_VALUE_DECIMAL)}
+                    {v.wind_speed != null && v.wind_speed.toFixed(MODEL_VALUE_DECIMAL)}
```

Comment:
So this is where you can use `{v.wind_speed?.toFixed(MODEL_VALUE_DECIMAL)}`
Sorry I only saw the first line

Reply:
And it will treat 0 or [] as a valid value